src/app/[locale]/seller/registration/page.tsx

```diff
@@ -388,6 +401,34 @@ const SellerRegistrationForm = () => {
           <ToggleCollapse
             header={t('SCREEN.BUY_FROM_SELLER.SELLER_CONTACT_DETAILS_LABEL')}
             open={false}>
+                <div className="mb-4">
+                <label>
+                  {t('SCREEN.SELLER_REGISTRATION.EMAIL_LABEL')}
+                </label>
+                <p className="text-gray-400 text-sm mt-1">
+                </p>
+                <Input
+                  name="email"
+                  type="text"
+                  value={formData.email}
+                  onChange={handleChange}
+                />
+              </div>
+              <div className="mb-4">
```

Comment:
Shouldn't we continue using the same input field type for the phone number as we had in the User Preferences sidebar? Note that we previously used the `TelephoneInput` component for that field.

Reply:
You are right about this one, it was an oversight from my part. Thanks for noticing it.